src/components/Explore/utils.ts

```diff
@@ -452,6 +457,21 @@ export const exploreMetricData: {
       },
     ],
   },
+  [ExploreMetric.DAILY_CASES_PER_100K]: {
+    title: 'Daily New Cases per 100k',
+    name: 'Daily New Cases per 100k',
+    chartId: 'daily_new_cases',
+    dataMeasure: DataMeasure.INTEGER,
+    yAxisDecimalPlaces: 0,
+    seriesList: [
+      {
+        label: 'Daily New Cases per 100k',
+        tooltipLabel: 'Daily New Cases per 100k',
```

Comment:
nit: I think for consistency with other metrics these should be lowercase other than the first letter.